libsql-sqlite3/src/vdbe.c

```diff
@@ -3719,7 +3734,7 @@ case OP_Count: {         /* out2 */
     nEntry = 0;  /* Not needed.  Only used to silence a warning. */
     i64 nPages = 0;
     rc = sqlite3BtreeCount(db, pCrsr, &nEntry, &nPages);
```

Comment:
@MarinPostma, not a subject of this PR, but I spotted a weird behaviour of the `sqlite3BtreeCount` - we don't count read pages in case of non success return code (for example, if query were interrupted). Should we fix it or this is desired behaviour?

Reply:
I don't really know, afaik the counter is not necessarily very accurate, sometimes it counts pages, sometimes it's rows. I think we should consider it to be an estimate of how many rows were written/read. From that perspective I don't think we care too much about accounting when there are errors. If somebody complains we'll look I to it I suppose, but we already saturating the counter on overflow so I don't think it matters too much to us 😄